docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,708 @@
+    "\n",
+    "Unfolded polypeptides have a very large number of degrees of freedom\n",
+    "and thus an enormous number of potential conformations. For example, a\n",
+    "chain with 100 amino acids has on the order of 10 conformations. In\n",
+    "reality, however, many proteins fold to their native structure within\n",
+    "seconds. This is known as Levinthal’s paradox [1].\n",
+    "\n",
+    "The exponential growth of potential conformations with chain length\n",
+    "makes the problem intractable for classical computers. In the quantum\n",
+    "framework, our resource-efficient algorithm scales linearly with\n",
+    "the number of amino acids N.\n",
+    "\n",
+    "\n",
+    "<font size='2'>[1] https://en.wikipedia.org/wiki/Levinthal%27s_paradox </font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3110941f",
+   "metadata": {},
+   "source": [
+    "The goal of this work is to determine the minimum energy configuration of a protein. Starting from a random configuration, the protein's structure is optimized to lower the energy. This can be achieved by encoding the protein folding problem into a qubit operator and ensuring that all physical constrains are satisfied. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29deb98b",
+   "metadata": {},
+   "source": [
+    "For the problem encoding we use: \n",
+    "\n",
+    "- Configuration qubits: qubits that are used to describe the configurations and the relative position of the different beads\n",
+    "- Interaction qubits: qubits that encode interacations between the different aminoacids\n",
+    "\n",
+    "For our case we use a tetrahydral lattice (diamond shape lattice) where we encode the movement through the configuration qubits (see image below). \n",
+    "\n",
+    "<img src=\"aux_files/lattice_protein.png\" width=\"300\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f571d4a8",
+   "metadata": {},
+   "source": [
+    "The Hamiltonian of the system for a set of qubits $\\mathbf{q}=\\{\\mathbf{q}_{cf}, \\mathbf{q}_{in}\\}$ is \n",
+    "\n",
+    "$$H(\\mathbf{q}) = H_{gc}(\\mathbf{q}_{cf}) + H_{ch}(\\mathbf{q}_{cf}) + H_{in}(\\mathbf{q}) $$\n",
+    "\n",
+    "where \n",
+    "- $H_{gc}$ is the geometrical constrain term (governing the growth of the primary sequence of aminoacids without bifurcations\n",
+    "- $H_{ch}$ is the chirality constrain (enforcing the righ stereochemistry for the system)\n",
+    "- $H_{in}$ is the interaction energy terms of the system. In our case we consider only nearest neighbor interactions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d99a5f",
+   "metadata": {},
+   "source": [
+    "Further details about the used model and the encoding of the problem can be found in [2].\n",
+    "\n",
+    "<font size='2'>[2] A.Robert, P.Barkoutsos, S.Woerner and I.Tavernelli, Resource-efficient quantum algorithm for protein folding, NPJ Quantum Information, 2021 </font>"
```

Comment:
I am not sure whether it is Github's rendering, but I do not see this in the output.

Reply:
I can see it locally. I guess is rendering.

Reply:
We need to be careful and check how the Sphinx docs render it..

Reply:
When the next CI passes you can download the generated docs as artefacts and take a look :+1:

Reply:
After CI has run download the 'tutorials3.8` Artifact from the CI build Details page and expand that. It has full html as would be published.